e2e/wdio/core/tests/select.e2e-spec.ts

```diff
@@ -0,0 +1,398 @@
+import { SelectPo } from '../pages/select.po';
+import {
+    addIsActiveClass,
+    checkElementScreenshot,
+    click,
+    focusElement,
+    getAttributeByName,
+    getElementArrayLength,
+    getImageTagBrowserPlatform,
+    getText, isElementClickable,
+    isElementDisplayed,
+    mouseHoverElement,
+    refreshPage,
+    saveElementScreenshot,
+    waitForElDisplayed,
+    waitForInvisibilityOf
+} from '../../driver/wdio';
+import {
+    addRemoveSelectOptions,
+    customControlSelectOptions,
+    extendedSelectOptions,
+    maxHeightSelectOptions,
+    mobileSelectOptions,
+    optionTag,
+    programmaticSelectOptions,
+    selectModesCompactSelectOptions,
+    selectModesDefaultSelectOptions,
+    semanticModesErrorSelectOptions,
+    semanticModesInformationSelectOptions,
+    semanticModesSuccesSelectOptions,
+    semanticModesWarningSelectOptions
+} from '../fixtures/testData/select-tags';
+
+describe('Select component:', function() {
+    beforeEach(() => {
+        refreshPage();
+    });
```

Comment:
please add the retry

Reply:
why beforeEach instead of afterEach? you will load the page first time and immediately refresh the page without doing anything.

and please move this to be after the const declarations

Reply:
Fixed